helm/minio/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Multi-Cloud Object Storage
 name: minio
-version: 3.5.6
+version: 3.5.7
 appVersion: RELEASE.2022-02-18T01-50-10Z
 keywords:
```

Comment:
do not change versions that is our work @FernFerret

Reply:
Sorry will resolve. Most of the other charts I've interacted with require chart versions before the PR can even be built.